src/Application.vue

```diff
@@ -7,13 +7,18 @@
 
 <script>
 import FirstComponent from './components/FirstComponent';
+import getAnalytics from './services/analytics';
```

Comment:
FirstComponent, rly?

Reply:
This is code that Nick left for us)
It will be removed when we start to create our components.